### PR TITLE
Desktop Version

### PR DESCRIPTION
**Pull request by sunga12**
Desktop version completed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,32 +59,43 @@ <h3 class="SD">I'm a Software Developer</h3>
...
-    </div>
+  </div>
 </section>
 
     <section class="Recent-works">
-
+      <section class="recent-works-heading">
         <h1 class="recent-works-title">My Recent Works</h1>
-      
+        <div class="line"></div>
```
Comment (RaoAkif) [inline index.html:68]: I have observed a few design issues, that could be eliminated using `dislay: none` when you transition from the mobile to the desktop version. To look at, how you can do that, you can follow [this guide.](https://www.w3schools.com/css/css_display_visibility.asp) 

1. So this line:
image

is appearing in the desktop version here:
image

2. This line
image

in the desktop version appears like this
image

But keep in mind, here in the `form` section, you do not have to use the `display: none` property, you have to switch from the small bold line to this
image

File: index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 </head>
 <body>
 <section class="Home"></section>    
-<header class="home-header">
+<header class="mob-home-header">
     <div class="name logo">
         <div class="name-text" ><a href="" id="logo">Sungabanja Thawethe</a></div>
     </div>
```
Comment (RaoAkif) [inline index.html]: [OPTIONAL] You can use your short name here. For example, my full name is `Rao Muhammad Akif Tufail`. But it does look quite expanded. So I use a short one like `Rao Akif`. This is an optional suggestion. If you feel OK with your short name, that's great. By the way, I have also seen some guys, making a text logo, that takes a square or a circular shape, which also looks fine. Think about what feels good for design.

File: index.html (hunk 1 above)
Comment (RaoAkif) [inline index.html]: When I looked at the desktop version, `My Recent Work` section
image
It is behind your `intro` section. Kindly try to meet the FIGMA design requirement.
The purpose of this project is to learn how to successfully switch your design from mobile to desktop design using media queries.

File: index.html [hunk 3]
```diff
@@ -135,13 +173,17 @@ <h2 class="professional">Professional Art Printing Data</h2>
...
     <h1 class="about-title">About me</h1>
-    <p class="about-me">Hello I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <p class="about-me">Hey there! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
     <button class="resume-link"><a href="#">Get My Resume</a></button>
-  
+    </section>
+
+      <div class="divider"></div>
+
     <section class="lists">
```
Comment (RaoAkif) [inline index.html]: This is from FIGMA design:
image


This is your current bullet points:
image

You just have tp export these icons, and apply them there.

File: index.html [hunk 4]
```diff
@@ -203,7 +246,8 @@ <h2 class="contact-comment">I'm always interested in hearing about new projects,
         </li>
         <li>
           <label for="msg"></label>
-          <textarea id="msg" name="user_message" required maxlength = "500" placeholder="Enter text here"></textarea>  
+
+          <textarea id="msg" name="user_message" required maxlength = "500" placeholder="Hello Sunga. I would like to get in touch with you..."></textarea>
```
Comment (RaoAkif) [inline index.html]: The underlines in the form inputs are missing. You can add them using `border-bottom`

This is your design:
image
This is FIGMA design:
image

**Review by RaoAkif — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️

Hi @sunga12,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Required Changes ♻️
#### Please remove the branch conflicts before taking another review, otherwise it would be marked as InValid
image

_Check the comments under the review._


## Optional suggestions
- Remove all the excess files from the project that you might be working on to match the design. If you want to work with them for testing or other reason, you can create another branch, or altogether another similar project locally, whatever you feel comfortable with.
- _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (sunga12) [reply index.html]: Hi @RaoAkif 

Kindly clarify this comment as I dont quite understand what you meant.

File: index.html (hunk 4 above)
Comment (sunga12) [reply index.html]: hi @RaoAkif 

I did try to use border-bottom , but each time i did, it brought, the entire border around the text, so I wasnt sure what else to do.

**Comment by sunga12** [comment]
Required changes to be made:

- Resolve all branch conflicts
- Remove all unnecessary files
- Reposition the Recent works page to eliminate overlap
- Reposition the Horizontal bar and display it correctly in both mobile and desktop version
- Use shorthand name for logo
- Change bullet lists on the about me page to match the Figma template
- Add a bottom border to form inputs

Changes made:

- Resolved all branch conflicts
- Removed all unnecessary files
- Repositioned the Recent works page to eliminate overlap
- Repositioned the Horizontal bar on the about me page and display it correctly in both mobile and desktop versions
- Used shorthand name for logo
- Changed bullet lists on the about me page to match the Figma template
- Added a bottom border to form inputs

File: index.html (hunk 1 above)
Comment (wayungi) [reply index.html:68]: - [  ] You have done very good work ✔️ work implementing this request. You however forgot one small detail about the change. Please assign the correct color to that dividing line. Check the Figma design. 👓  The color is 
 ```
background: #DFE1E6;
```

_**Figma**_
border


 **Your design**
 
border-2

File: index.html [hunk 5]
```diff
@@ -62,7 +62,7 @@ <h3 class="SD">I'm a Software Developer</h3>
   </div>
 </section>
 
-    <section class="Recent-works">
+    <section class="recent-works">
```
Comment (wayungi) [inline index.html:65]: - [ ] Please Fix the icons. You need to add a `.` to the beginning of the icon file path as shown below `<a href="#"></a>` You should always start your relative path with a `.` You can read more about it [here](https://stackoverflow.com/questions/66887694/in-the-file-path-what-does-mean) 
links-1

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
image
- [ ] Good PR title
- [ ] GitHub flow
- [ ] Descriptive commit message
- [ ] Professional README
- [ ] All linter checks passing


 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sunga12** [comment]
Changes Required

- Change the path for social media icons
- Change colour for the horizontal line in the contact page to DFE1E6

Changes made:

- Changed the path for social media icons
- Changed colour for the horizontal line in the contact page to DFE1E6

File: index.html [hunk 6]
```diff
@@ -62,7 +62,7 @@ <h3 class="SD">I'm a Software Developer</h3>
...
       <section class="recent-works-heading">
         <h1 class="recent-works-title">My Recent Works</h1>
         <div class="line"></div>
```
Comment (AlphaNtihinduka) [inline index.html:68]: - [ ] [OPTIONAL] Please ensure that you follow the Figma design template. I have noticed that the `padding-top` is large in the recent section. Kindly ensure to reduce that to make the project look professional ☺️ 

Your design 😎 
up rece1

Figma design 😎 
up rece2

File: styles.css [hunk 7]
```diff
@@ -1246,7 +1248,7 @@
   }
 
   .proj-desc {
```
Comment (AlphaNtihinduka) [inline styles.css:1250]: - [  ] [OPTIONAL] I have noticed that the `project description` is overlapping the subtitle 😞 . Kindly ensure that the description is independent.

Your design 😎 
overlapping

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @sunga12 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

### Danger Zone ❌ 
- [ ] [OPTIONAL] However much this is marked optional, kindly ensure to implement the suggestions before proceeding 🚶‍♂️ . I have noticed that there is a horizontal scroll on your webpage. The experience of scrolling horizontally can make the screen judder and create a headache-inducing feeling 😞 . 
resp

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**